snapcraft.yaml

```diff
@@ -17,4 +17,16 @@ parts:
       - grub-pc-bin
       - grub-common
     plugin: make
+  uuidgen:
+    plugin: nil
+    stage-packages: [ uuid-runtime ]
+    stage: [ usr/bin/uuidgen, usr/share/doc/uuid-runtime/copyright ]
 
+hooks:
+  prepare-device:
+    environment:
+    # TODO: define your model's API key here
+    # See https://ubuntu.com/core/services/guide/serial-vault-overview
+    # for instructions on how to generate an API key
+    # DO NOT check this API key into a publicly accessible VCS
```

Comment:
Nitpick^2: I would prefer this comment indented to the right a couple of spaces.

Reply:
I think this is what you were asking, but let me know if it should go further